### ISSUE TITLE
Hook::all() returns empty array inside Command#*Action()

### ISSUE THREAD

**Issue by Al2Klimov** [Icinga/icingaweb2#3632]
## Expected Behavior

`Hook::all()` returns the same array inside `Controller#*Action()` and `Command#*Action()`.

## Current Behavior

`Hook::all()` returns filled array inside `Controller#*Action()`, but not inside `Command#*Action()`.

## Possible Solution

Fix it.

## Steps to Reproduce (for bugs)

1. Run...

```
base64 -D <<EOF |tar -vxz
H4sIAHtB9FsAA+2ZTW+bQBCGfeZX7KFSnEit+VykpGob+dJIrZRbLkgVAVKjAIv4cFNV+e9d
vPgjtBQ2IRspfZ8Li70L2O/MzgyTsrBOonIRJPGKsduAZWWdRsViNiE6x3UccaTiqJu2OApm
hmW5jm45Lh/rhmNSc0acKR+ij7qs/IKQmX+bxClb987j025uVDyQWtIe/ZP4uvCLn5PYgYz+
lk65/tS1dOivgiH9lxPsC9L6m6bu2tBfBbL6f+YnskYgoT/VHbPR33Ut6K+CR+n/hSVLv2qG
7/JVPniPRmBq2736G4bV0d+xXTojuoLf/7/r//4jV1DTMj+NytwPInIRxNl33/u6sQuvI7/X
aH6maUHilyXZm4H2S7vXXvqngEfQ5/9+nidx4Fcxy56cA8rs/9z1ef7nOi7yfyWM0Z9/F7A0
9bOwfO78zzUdkf9RCv1VIKv/JQuWYjwq9G8Yiv82F/uh/jbfBxD/VSAZ/5d7U+BpQF3u5p/v
7aVNEg6+5Ku81mp2ycPekEh0V0X8gqQ958kE4eT1Nb8iuamzoLkqKVfsx/lmOD/eTBDTGtZ+
8S2s03ze3Pn01E+S+VE3cRXJytHx8dluWXQXV/OA1Vk1amG78v5VpTqj/J9lVcGSJCqeff/f
1n+GhfpfCbL6X2RhdLfcfTBJ/Wcbdrf+4wkg9n8VyO7/e1OQ2P+vouuDlbsQ0LGlgziw/agn
FMTNwkliQRhHr3FXH0/H//OCrePw5fo/wv/R/1FGn/4v1f8xqCv6P4j/ShjSfznBviClPy/8
Nv0fA/qrQFb/SzEIZfpAMvqbRpP/2TZF/0cJT9J/ZHNYvv5zqYX4r4RJ9Rd59R814VD9Z9r0
of4WTwAt1H8qGFf/beX3DuXvFocPC8J/NBC9feOw00vclYBoLSqiz/+LOhv/gn+AIf83nE7/
36AO+v9qaP1/cXJCPq35H+H95X2OcOWyPZI31SouyclC0zajtx9ao2m8dX7UfZG4fddyBicG
AAAAAAAAAAAAAAAAAAAAAAAAAAAAgCn5DYrP6QMAUAAA
EOF
```

... which will extract...

```
.
└── modules
    ├── clihookconsumer
    │   ├── application
    │   │   ├── clicommands
    │   │   │   └── PocCommand.php
    │   │   └── controllers
    │   │       └── IndexController.php
    │   └── library
    │       └── Clihookconsumer
    │           └── Hook
    │               └── LolCatHook.php
    └── clihookprovider
        ├── library
        │   └── Clihookprovider
        │       └── ProvidedHook
        │           └── Clihookconsumer
        │               └── LolCat.php
        └── run.php
```

2. Enable the modules clihookconsumer and clihookprovider.
3. Run `icingacli clihookconsumer poc show` and compare to  /icingaweb2/clihookconsumer.

## Context

A secret.

## Your Environment

The latest bleeding edge hipster stuff from you.

**Comment by Al2Klimov** [comment]
I'm afraid the problem is quite bigger:

```
$ tree hook-cli-test 
hook-cli-test
├── authentication.ini
├── config.ini
├── enabledModules
│   ├── clihookconsumer -> /Users/aklimov/NET/WS/icingaweb2/modules/clihookconsumer
│   └── clihookprovider -> /Users/aklimov/NET/WS/icingaweb2/modules/clihookprovider
└── roles.ini
$ ICINGAWEB_CONFIGDIR="$(pwd)/hook-cli-test" REMOTE_USER=icingaadmin ~/homebrew/Cellar/php@7.0/7.0.30/bin/php bin/icingacli mod li
MODULE         VERSION   STATE     DESCRIPTION
clihookconsumer 0.0.0     enabled   clihookconsumer
clihookprovider 0.0.0     enabled   clihookprovider
$ while ICINGAWEB_CONFIGDIR="$(pwd)/hook-cli-test" REMOTE_USER=icingaadmin ~/homebrew/Cellar/php@7.0/7.0.30/bin/php bin/icingacli clihookconsumer poc show; do git checkout HEAD^; done # the command does exit(count(Hook::all('Clihookconsumer/LolCat')));
(...)
Previous HEAD position was b643c97fb... IdoQuery: pass ourselves to hook, not back to us
HEAD is now at 30f8faf0b... Merge branch 'feature/hook-api-10613'
array(0) {
}
Previous HEAD position was 30f8faf0b... Merge branch 'feature/hook-api-10613'
HEAD is now at 8f8d251fd... Use old class loader for tests (WIP)
PHP Fatal error:  Uncaught Error: Class 'Icinga\Application\Hook' not found in /Users/aklimov/NET/WS/icingaweb2/modules/clihookconsumer/application/clicommands/PocCommand.php:12
Stack trace:
(...)
  thrown in /Users/aklimov/NET/WS/icingaweb2/modules/clihookconsumer/application/clicommands/PocCommand.php on line 12

Fatal error: Uncaught Error: Class 'Icinga\Application\Hook' not found in /Users/aklimov/NET/WS/icingaweb2/modules/clihookconsumer/application/clicommands/PocCommand.php:12
Stack trace:
(...)
  thrown in /Users/aklimov/NET/WS/icingaweb2/modules/clihookconsumer/application/clicommands/PocCommand.php on line 12
$ git describe 
v2.0.0-335-g8f8d251fd
```

**Comment by Al2Klimov** [comment]
`Icinga::app()->getModuleManager()->loadedAllEnabledModules();` doesn't help either.

**Comment by nilmerg** [comment]
Closed in favor of #3771.
